app/models/miq_expression/field.rb

```diff
@@ -39,6 +37,14 @@ def target
     end
   end
 
+  private
+
+  delegate :arel_table, :to => :target
+
+  def arel_attribute
```

Comment:
If you have access to the model, maybe you could use `model.arel_attribute(column)` instead?